### PR TITLE
draft: Split MCE into modules

### PR DESCRIPTION
**Pull request by Rushaway**
Any reviews @maxime1907 ?

### CODE REVIEW

**Comment by Rushaway** [comment]
I added a prefix because I plan to do the NE aswell with the same structure. It was for maybe avoid mistake, should i remove the prefix anyway?

**Comment by maxime1907** [comment]
yeah you can just create a "ne" folder and do the same so you can remove prefixes

File: addons/sourcemod/scripting/mapchooser_extended.sp [hunk 1]
```diff
@@ -60,33 +59,20 @@
...
-	GameType_Custom		= 3,
-}
-
-enum
-{
-	GunGameMode_ArmsRace	= 0,
-	GunGameMode_Demolition	= 1,
-	GunGameMode_DeathMatch	= 2,
-}
+#include "mce_globals_variables.inc"
```
Comment (maxime1907) [inline addons/sourcemod/scripting/mapchooser_extended.sp:67]: mce/

**Comment by maxime1907** [comment]
mapchooser_extended.sp(74) : error 417: cannot read from file: "mce/natives.inc"

**Comment by Rushaway** [comment]
> mapchooser_extended.sp(74) : error 417: cannot read from file: "mce/natives.inc"

https://github.com/srcdslab/sm-plugin-mapchooser-extended/actions/runs/10177790686/job/28150026846?pr=23

**Comment by maxime1907** [comment]
you forgot to rename natives

**Pull request merged**